Offroad-iOS/Offroad-iOS/Global/Components/SegmentedControl/ORBSegmentedControl.swift

```diff
@@ -18,16 +18,14 @@ final class ORBSegmentedControl: UIView {
     
     //MARK: - Properties
     
-    let animator = UIViewPropertyAnimator(duration: 0.3, dampingRatio: 1)
-    weak var delegate: ORBSegmentedControlDelegate? = nil
-    var titles: [String]
-    var isLayoutDone: Bool = false
+    private let underbarAnimator = UIViewPropertyAnimator(duration: 0, dampingRatio: 1)
+    private lazy var underbarLeadingConstraint = underbar.leadingAnchor.constraint(equalTo: leadingAnchor)
+    private var segmentWidth: CGFloat { frame.width / CGFloat(titles.count) }
+    var segmentsCount: Int { stackView.arrangedSubviews.count }
```

Comment:
P3

`init` 함수 내부에서 `titles` 배열을 통해 `stackView`의 `arrangedSubviews`를 안전하게 동기화해주고 있지만, 혹시 모를 상황이나 유지보수 측면에서 생각하면 하나의 요소를 기준으로 잡는 것이 좋다고 생각합니다!

따라서 `segmentsCount` 값 자체를 UI 요소 `stackView`보다는 데이터 요소인 `titles`의 count 기준으로 통일시키는 것이 좋아보이는데 민성님은 어떻게 생각하시는지 궁금합니다~!

Reply:
@Johyerin 
말씀하신 부분에 동의합니다.
`titles`의 `count`를 기준으로 계산하고
더 나아가서 아예 segmentWidth 계산 속성 내에서 segmentsCount 속성을 사용하는 것도 괜찮을 것 같은데, 어떻게 생각하시나요?
```suggestion
    private var segmentWidth: CGFloat { frame.width / CGFloat(segmentsCount) }
    var segmentsCount: Int { titles.count }
```